src/components/Prank/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from "react";
+import Cookies from "js-cookie";
+import { Button } from "components/Button";
+import { Modal } from "components/Modal";
+import { StyledLink } from "components/styles";
+import { useToastContext } from "context/toast";
+
+const START_PRANK = new Date("April 1, 2021 11:00:00");
+const END_PRANK = new Date("April 2, 2021 00:00:00");
+const COOKIE_NAME = "PRANK_EXPERIENCED";
+
+export const Prank = () => {
+  const [isVisibleModal, setIsVisibleModal] = useState(false);
+  const { info } = useToastContext();
+  const now = new Date();
+  const isTodayAprilFoolsDay = now >= START_PRANK && now < END_PRANK;
+  const prankExperienced = Boolean(Cookies.get(COOKIE_NAME));
+  const shouldRunPrank = !prankExperienced && isTodayAprilFoolsDay;
+
+  useEffect(() => {
+    if (shouldRunPrank) {
+      setTimeout(
+        () =>
+          info(
+            <>
+              <StyledLink
+                onClick={() => {
+                  //   Cookies.set(COOKIE_NAME, "true");
```

Comment:
Accidentally left this commented. But i think its fine if people want to see it again its easy to do so and its easily dismissible.